src/features/beta-feedback/BetaFeedbackBadge.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import classNames from 'classnames';
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import { BetaBadge } from '../../components/badge';
+import { Link } from '../../components/link';
+import Tooltip from '../../components/tooltip/Tooltip';
+
+import './styles/BetaFeedbackBadge.scss';
+
+type Props = {
+    className?: string,
+    formUrl: string,
+    tooltip?: boolean,
+};
+
+const BetaFeedbackBadge = ({ className = '', tooltip = false, ...rest }: Props) => {
+    const classes = classNames('bdl-HeaderFeedbackBadge', className);
+    const { formUrl } = rest;
```

Comment:
Any reason why you're unpacking formUrl from `rest` instead of naming it like the other props?

Reply:
I initially wanted to put just the defaulted params on L18, and unpack the rest from `rest`, but we ended up only having the one const like this. Personally, I find the `({ ...}: Props) => {` a bit hard to read when it gets long or spans multiple lines.